src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -258,6 +285,22 @@ def get_or_create(cls, region: Region) -> Optional["Proxy"]:
                     proxy.state = VmState.stopping
                     proxy.save()
                 return None
+            if proxy.timestamp is not None:
+                proxy_timestamp = proxy.timestamp
+                if proxy_timestamp < (
+                    datetime.datetime.utcnow() - datetime.timedelta(7)
```

Comment:
I don't think this is what you want; `utcnow()` returns a naive `datetime`: https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow

See here for prior art, where we return a TZ-aware `datetime`: https://github.com/microsoft/onefuzz/blob/e238542800fdd855bac8c3593d06028bedfe5c99/src/api-service/__app__/onefuzzlib/proxy.py#L223